manifests/overlays/odhdashboardconfig/odh-dashboard-config.yaml

```diff
@@ -40,3 +40,6 @@ spec:
       requests:
         cpu: "4"
         memory: 4Gi
+  groupsConfig:
+    adminGroups: 'odh-admins'
```

Comment:
i am not sure if this should be rhods-admin , is it correct ?

Reply:
I think it should be dedicated-admins as for previous rhods versions

Reply:
Just so you are aware. The manifests in this repo are not the final manifest that is used for a RHODS deployment.  The official odh-dashboard deployment files will still be located in https://github.com/red-hat-data-services/odh-manifests/tree/master/odh-dashboard AND the RHODS runtime configurations for the ODHDashboardConfig will be deployed via https://github.com/red-hat-data-services/odh-deployer.